go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/talos-systems/talos-controller-manager
 
 go 1.13
 
+replace github.com/talos-systems/talos => /vagrant/code/github.com/talos-systems/talos
```

Comment:
looks like this shouldn't be here ;)

Reply:
good catch ... updated to use my fork until the PR in Talos is merged.